compositCommon/AcquamanMPlot.pri

```diff
@@ -2,9 +2,14 @@
 
 macx {
 		# EPICS Dependencies:
-		EPICS_INCLUDE_DIRS = $$PROJECT_ROOT/acquaman/contrib/epics/base/include \
-					$$PROJECT_ROOT/acquaman/contrib/epics/base/include/os/Darwin
-		EPICS_LIB_DIR = $$PROJECT_ROOT/acquaman/contrib/epics/base/lib/darwin-x86
+		EPICS_INCLUDE_DIRS = $$PATH_TO_ACQUAMAN/../contrib/epics/base/include \
```

Comment:
@davidChevrier we need to add some user name scoping for this since you just removed the code to get my mac to run the data server.